.github/workflows/ci-docs-shim.yaml

```
@@ -2,23 +2,6 @@ name: CI Docs Shim
 
 on:
   pull_request:
```

```suggestion
  pull_request:
    branches: [main]
    types: [milestoned, opened, synchronize]
```
This should match test IMO so we can do the milestoned thing if we need to.